Common subdirectories: btrfs-progs-v3.17.3_alt/Documentation and btrfs-progs-v3.17.3/Documentation
diff -u btrfs-progs-v3.17.3_alt/Makefile btrfs-progs-v3.17.3/Makefile
--- btrfs-progs-v3.17.3_alt/Makefile	2014-12-04 14:47:18.000000000 +0100
+++ btrfs-progs-v3.17.3/Makefile	2014-12-06 17:21:50.630051000 +0100
@@ -26,7 +26,7 @@
 INSTALL = install
 prefix ?= /usr/local
 bindir = $(prefix)/bin
-lib_LIBS = -luuid -lblkid -lm -lz -llzo2 -L.
+lib_LIBS = -lattr -luuid -lblkid -lm -lz -llzo2 -L.
 libdir ?= $(prefix)/lib
 incdir = $(prefix)/include/btrfs
 LIBS = $(lib_LIBS) $(libs_static)
Common subdirectories: btrfs-progs-v3.17.3_alt/tests and btrfs-progs-v3.17.3/tests
